src/main/java/lt/akademija/itacademymanager/model/Review.java

```diff
@@ -0,0 +1,138 @@
+package lt.akademija.itacademymanager.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+
+@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Table(name = "review")
+public class Review {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer id;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "studentId")
+    private Student student;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "streamId")
+    private Stream stream;
+
+    private int overallGrade;
+    private String overallComment;
```

Comment:
O mums jo tikrai reikia? as kiek isivaizduoji mes ji tiesiog apskaiciuosim kaip vidurki is kitu skillu.

Reply:
aš manau, kad reikia, nes subjektyviai vertinant nebūtinai bendras įvertinimas atitiks vidurkį. Pvz studentas visur silpnokai įvertintas, bet overall dėl potencialo ar kokios kitos formoje neatsispindinčios priežasties dėstytojas jį vertina aukščiau ir overall komentaruose pasako kodėl.